client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java

```diff
@@ -704,7 +704,7 @@ public void operationFail(Throwable throwable) {
                         onExceptionImpl(brokerName, msg, timeoutMillis - cost, request, sendCallback, topicPublishInfo, instance,
                             retryTimesWhenSendFailed, times, ex, context, true, producer);
                     } else {
-                        MQClientException ex = new MQClientException("unknow reseaon", throwable);
+                        MQClientException ex = new MQClientException("unknown reason", throwable);
```

Comment:
If the exception here is a RemotingTooMuchRequestException, you should also stop retrying.

Reply:
You are right, and the invokeAsync won't throw RemotingTooMuchRequestException, I will make some changes.